src/themes/elife/custom-properties.css

```diff
@@ -0,0 +1,9 @@
+:--root {
+  /* Colors */
+  --color-text-secondary: #888;
+
+  /* Typography */
```

Comment:
With the comments I was just bringing them over from the Pattern Librtary hence why they are there.

Reply:
@davidcmoulton With the spacing on the author list does this need to be a separate PR as the spacing would be on the bottom of the Author List rather than the top of the date?

Reply:
@discodavey Happy to have a separate PR as part of the same ticket.